tests/test_authentication.py

```diff
@@ -23,14 +23,14 @@ def setup_class(self):
         ):
             self.cognito_user = {
                 "cognito:username": str(uuid4()),
-                "custom:id": str(uuid4()),
                 "email": f"{str(uuid4())}@{str(uuid4())}.com",
+                "custom:id": str(uuid4()),
                 "custom:1": str(uuid4()),
                 "custom:2": str(uuid4()),
+                "aud": allowed_audiences[0],  # moved due pylint and minimising scope of PR
```

Comment:
This comment makes sense only in the pull request where I see exactly what was introduced, but in the code, you should rather avoid things like this or point directly to [#214](https://github.com/PyCQA/pylint/issues/214).